xbout/load.py

```diff
@@ -302,6 +308,8 @@ def attrs_remove_section(obj, section):
         if info:
             print("Applying {} geometry conventions".format(geometry))
 
+        if _is_dir(gridfilepath):
+            gridfilepath += "/" + ds.options["grid"]
```

Comment:
Probably worth checking whether `ds.options["grid"]` exists. `ds.options` will be `None` if no input file was passed, and some input files might not contain a `grid` options (although it would be odd for someone to be passing a `gridfilepath` argument if there was no grid...).

Reply:
Another option would be to just ignore in that case. Then I can always provide that, independent of whether the simulation has been using a grid or not, as long as all my grids are in one spot.

But I don't really care ...

Reply:
That is a good reason to not error. I'd be fine with just warning in this case, and continuing with no grid file.